### ISSUE TITLE
Question: How were WCAG techniques destructured in `wcag21.json`

### ISSUE THREAD

**Issue by arthurgousset** [w3c/wai-wcag-quickref#291]
Hey, I really like that `wcag21.json` has JSON formatted techniques, because it makes it easy to look up techniques associated with a success criterion. For example, I can answer questions like "What are the failure conditions for success criteria 1.1.1?".

```json
"techniques": [{
      "id": "TECH:G94",
      "title": "Providing short text alternative for non-text content that serves the same purpose and presents the same information as the non-text content",
```

```json
  {"failure": [
      {
          "id": "TECH:F3",
          "title": "Failure of Success Criterion 1.1.1 due to using CSS to include images that convey important information"
      },
```

Source: https://github.com/w3c/wai-wcag-quickref/blob/gh-pages/_data/wcag21.json#L62-L68

`wcag22.json`, unfortunately, has HTML formatted techniques: 

```html
"techniquesHtml": {
      "sufficient": [
        {
          "title": "Situation A: If a short description can serve the same purpose and present the same\n               information as the non-text content:",
          "content": "<ul>\n               <li>\n                  <a href=\"https://www.w3.org/WAI/WCAG22/Techniques/general/G94\">G94: Providing short text alternative for non-text content that serves the same purpose and presents the same information as the non-text content</a> using one of the following \n                  <strong>\n                     <a href=\"#text-equiv-all-situation-a-shorttext\">Short text alternative techniques for Situation A</a>\n                  </strong>:\n       
```

I can see `wcag22.json` is updated and written using the script in `_scripts/update-wcag-json.mjs`, which parses whatever is returned by `https://www.w3.org/WAI/WCAG$VERSION/wcag.json`.

Question: Is it possible to JSON structure the techniques in `wcag22.json` as well? 

Thanks!

> [!NOTE]
> I would have asked this question a Github discussion, since it's not a bug, but discussions are not enabled on this repo.

**Comment by kfranqueiro** [comment]
Apologies for seeing this rather late. This overlaps with one of the items in w3c/wcag#4393, which has work in progress to resolve.

**Comment by arthurgousset** [comment]
> Apologies for seeing this rather late. This overlaps with one of the items in [w3c/wcag#4393](https://github.com/w3c/wcag/issues/4393), which has work in progress to resolve.

Ah sweet, thanks for sharing. I'll close this issue in favour of https://github.com/w3c/wcag/issues/4393 👍
